### PR TITLE
Glean ping schemas

### PR DESCRIPTION
**Pull request by fbertsch**
@mdboom to review the glean MSG changes
@whd to review the script and allowlist updates

Note: The allowlist does need to be updated since it deletes BQ schemas not in it, so although MGS adds them automatically, they still have to be included (I was wrong earlier).

### CODE REVIEW

File: allowlist [hunk 1]
```diff
@@ -1,6 +1,8 @@
 telemetry/core
 org-mozilla-fenix/.*
 org-mozilla-reference-browser/.*
+org-mozilla-samples-glean/.*
+org-mozilla-tv-firefox/.*
 activity-stream/.*
```
Comment (whd) [inline allowlist]: It is my understanding that `glean/.*` is not intended to be added to the allowlist, is that correct?

File: bin/schema_generator.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ set -exuo pipefail
 MPS_REPO_URL="git@github.com:mozilla-services/mozilla-pipeline-schemas.git"
 MPS_BRANCH_SOURCE="dev"
 MPS_BRANCH_WORKING="local-working-branch"
-MPS_BRANCH_PUBLISH="generated-schemas"
+MPS_BRANCH_PUBLISH="test-generated-schemas"
```
Comment (whd) [inline bin/schema_generator.sh]:
```suggestion
MPS_BRANCH_PUBLISH="generated-schemas"
```

File: allowlist (hunk 1 above)
Comment (fbertsch) [reply allowlist]: Right. Glean is a library that sends no telemetry on its own.

File: bin/schema_generator.sh (hunk 2 above)
Comment (fbertsch) [reply bin/schema_generator.sh]: Good catch. I'll open an issue about making this configurable for dev testing.

**Pull request merged**